lib/helper-functions.js

```diff
@@ -61,14 +61,14 @@ function componentNameToJavaScriptModuleName (componentName) {
  * the project root node_modules
  *
  * @param {string} packageName - Installed npm package name
- * @param {...string} [paths] - Optional child directory paths
+ * @param {string} [childPath] - Optional child directory path
  * @returns {string} Path to installed npm package
  */
-function packageNameToPath (packageName, ...paths) {
+function packageNameToPath (packageName, childPath) {
```

Comment:
Compiler said we can't spread `...` an optional parameter

We weren't using it anyway 😬